docs/pages/versions/v52.0.0/sdk/notifications.mdx

```diff
@@ -455,23 +467,25 @@ This value will be one of the following, accessible under `Notifications.IosAuth
 
 ## Notification events listeners
 
-Notification events include incoming notifications, interactions your users perform with notifications (this can be tapping on a notification,
-or interacting with it via [notification categories](#manage-notification-categories-interactive-notifications)), and rare occasions when your notifications may be dropped.
+Notification events include incoming notifications, interactions your users perform with notifications (this can be tapping on a notification, or interacting with it via [notification categories](#manage-notification-categories-interactive-notifications)), and rare occasions when your notifications may be dropped.
+
+Several different listeners are exposed, and documented in [Push Notification Behaviors](/push-notifications/what-you-need-to-know/#push-notification-behaviors) section.
```

Comment:
```suggestion
Several listeners are exposed and documented in the [Push notification behaviors](/push-notifications/what-you-need-to-know/#push-notification-behaviors) section.
```

Let's apply the same changes from unversioned in the SDK 52 reference for consistency.